boards/BMS/Src/App/App_Accumulator.c

```diff
@@ -71,11 +75,14 @@ struct Accumulator *App_Accumulator_Create(
     struct Accumulator *accumulator = malloc(sizeof(struct Accumulator));
```

Comment:
Need to send cmd out when the chip is awake. Alternatively we can wake up the chip, then send the enable/disable balancing commands to the chip. However, this would increase the time spent sending the msg on the SPI bus